src/views/dialog.rs

```diff
@@ -244,6 +249,11 @@ impl Dialog {
         Box::new(self.buttons.iter_mut().map(|b| &mut b.button.view))
     }
 
+    /// Returns currently focused element
+    pub fn focus(&self) -> &DialogFocus {
```

Comment:
Since `DialogFocus` is so small (it's barely lareger than a `usize`), and it'll be `Copy`, let's directly return a `DialogFocus` here without references. It'll prevent borrowchecker confusions later on... :)

Reply:
done